include/openthread-types.h

```diff
@@ -215,6 +215,19 @@ enum
     OT_SECURITY_POLICY_BEACONS                = 1 << 3,  ///< Beacons enabled
 };
 
+#define OT_CHANNEL_MASK_MAX_SIZE           4   ///< Maximum size of Channel Mask
+
+/**
+  * This structure represents Channel Mask.
+  *
+  */
+typedef struct otChannelMask
+{
+    uint8_t mChannelPage;
```

Comment:
I think this is moving in the right direction.  After thinking about this a bit more, I think we should actually consider simplifying this all the way down to a single `uint32_t` for the bit mask.  We can call this something like `otChannelMaskPage0`.  We can then reuse the same macros for setting the bits (e.g. `OT_CHANNEL_11_MASK`).

In the future, if Thread decides to support other channel pages, then we can add additional fields to the `otOperationalDataset` structure.